weblogic-azure-vm/arm-oraclelinux-wls-cluster/arm-oraclelinux-wls-cluster/src/main/arm/mainTemplate.json

```diff
@@ -615,6 +613,9 @@
                     "adminUsername": {
                         "value": "[parameters('adminUsername')]"
                     },
+                    "adminVMNamePrefix": {
+                    	 "value": "[parameters('adminVMNamePrefix')]"
+                    },                    
```

Comment:
In the `outputs` section, we do need to output the remote console URL for HTTP and HTTPS, exactly as we do in WLS on AKS.

See https://github.com/oracle/weblogic-azure/blob/556ebc6bfd92679ceeb843f0a1bdea98a06ca608/weblogic-azure-aks/src/main/bicep/mainTemplate.bicep#L801

This is very important for usability. Without it, it's very to demonstrate the utility of the product in a pre-sales or conference meeting.